cli/range.go

```diff
@@ -74,6 +72,7 @@ func runLsRanges(cmd *cobra.Command, args []string) {
 				i, replica.NodeID, replica.StoreID)
 		}
 	}
+	fmt.Printf("%d results\n", len(rows))
```

Comment:
I used 4 lines of code to get the proper pluralization of results. Irritating, yes, but `1 results` looks odd and wrong to me.